app/views/char_types/index.html.erb

```diff
@@ -43,7 +43,8 @@
             </tr>
           </table>
         <% else %>
-          <p>This character type has no attributes! <%= link_to 'Add some', new_char_type_char_attribute_path(type) %>.</p>
+          <p>This character type has no attributes! <%= link_to 'Add some', 
```

Comment:
I think this is better as one line, either that or put the whole link in the next line.